build.gradle

```diff
@@ -268,6 +268,9 @@ combineUnpickDefinitions {
 java {
 	sourceCompatibility = JavaVersion.VERSION_17
 	targetCompatibility = JavaVersion.VERSION_17
+	toolchain {
+		languageVersion.set(JavaLanguageVersion.of(17))
+	}
```

Comment:
No, this is a sperare issue and likely not the solution, Enimga should run just fine on J17+

Reply:
Well that is weird, then, as I have OpenJDK21 as default, but somehow it breaks with the following message:

```
> Could not open proj generic class cache for build file '/****/yarn/build.gradle' (/home/nerjal/.gradle/caches/8.1.1/scripts/89g4wxh1rs3okpbpslwfmo3gm).
   > BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 65
```

Reply:
Although turns out that change does not fix anything anyway, as it would appear I had forgotten I had set the gradle java home in my global `gradle.properties` as a fix attempt, but did not revert it. The build script change does not fix anything.